### ISSUE TITLE
unable to find certificate in default keystore for validation

### ISSUE THREAD

**Issue by artlessknave** [karakun/OpenWebStart#503]
![image](https://user-images.githubusercontent.com/579198/157774441-cca0d5cc-e873-4d58-aae3-f586be105d72.png)

is this fixable? it's for an x8 supermicro board. it's particularly important but was wondering.

**Comment by CoolxFrog** [comment]
I have the same problem.
OSX 12.6 - JVM 11.0.15

**Comment by sclassen** [comment]
Use the itw-settings executable which is bundled with OpenWebStart.
It allows you to brows you systems certificates and also add custom ones.

**Comment by artlessknave** [comment]
I don't have a certificate for these. these are ancient supermicro x8 boards.
I know generally what certificates are but I don't understand how to use them.
all I need is "I dont care about the certificate" does that exist? I don't see anything that looks like that in the itw-settings
there is a "server whitelist" section but it cannot be modified?

**Comment by sclassen** [comment]
There is no "I dont care about the certificate" option as WebStart is about executing code you download from the network.
Most of the pop up dialogs allow you grant a security exception. If you have a dialog which does not allow to do so then please add a screenshot here.

**Comment by artlessknave** [comment]
the screenshot is already above. there are no options, it just fails. they are old boards so it's not super important but I'm probably not the only person with them

**Comment by artlessknave** [comment]
![image](https://user-images.githubusercontent.com/579198/201786078-15c03fdb-c824-4637-a5b4-0d6b48dec086.png)
![image](https://user-images.githubusercontent.com/579198/201786136-6c84dcc8-340e-4338-8d10-932a0b6859d4.png)

**Comment by sclassen** [comment]
The first dialog with the options "yes" and "no" is from WebStart.
The second one is coming directly from your JVM. Nothing we can do here

**Comment by artlessknave** [comment]
after an update, i get this, which at least tells me something. I'm not sure if i can get past it though. i don't care that it isn't signed, this is very old java, quite possibly existed before certificates, or at least any certs it had would be very, very old. i just want it to run. can that be done, or is this an impossible dream?

Launch exception
netx: Initialization Error: Could not initialize application. (Fatal: Application Error: Cannot grant permissions to unsigned jars. Application requested security permissions, but jars are not signed.)
App is signed: false

**Comment by schmitmd** [comment]
yeah, based on #181 , this is impossible.  Really too bad because, like @artlessknave, I can't use the Jviewer console for my super old x8 server at all.  I'm having a chicken-and-egg issue where I just need to (re)set the BIOS to the proper SoL settings but in order to do so, I need to boot up into it and see what I'm doing.  Only way I know to do so without physical access is the crappy Jviewer that hasn't seen an update in a very long time (and will never see one again)  _sigh_

**Comment by schmitmd** [comment]
As a follow-up, I was able to get it working by using this project: https://github.com/sciapp/nojava-ipmi-kvm
Caveat: You need to have an SSL certificate installed and HTTPS must be functional, even if it's an old/self-signed cert.  No HTTP-only unless you build the Docker image yourself and mess w/ the entrypoint.sh (I didn't have the patience to try that, but ymmv).

**Comment by nshopik** [comment]
Only workaround I found is using - Java 7 Update 45 installed in VM and security set to lowest in java settings (to allow run unsigned applets)
